webapp/packages/plugin-data-viewer/src/TableViewer/TableViewer.tsx

```diff
@@ -192,6 +192,19 @@ export const TableViewer = observer<Props>(function TableViewer({
     onValuePresentationChange,
   }, ['setPresentation', 'setValuePresentation', 'switchValuePresentation', 'closeValuePresentation']);
 
+  const needRefresh = getComputed(() => (
```

Comment:
Please cast dataModel to a boolean !!dataModel  so needRefresh will return boolean, not boolean | undefined